src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -1,128 +1,301 @@
+1. <a name="step-8"></a>Click **Create security group** to create a security group within your VPC that allows inbound access from your application or source program on Port 26257: {{ security_group_substeps }}
+
+Use either the Amazon VPC Console or the [AWS Command Line Interface (CLI)](https://aws.amazon.com/cli/) to continue:
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="serverless">
 
-### Enable private DNS
+<div class="filters clearfix">
+  <button style="width: 15%" class="filter-button" data-scope="aws-console">AWS Console</button>
+  <button style="width: 15%" class="filter-button" data-scope="aws-cli">AWS CLI</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="aws-console">
 
-1.  On the Amazon VPC Console **Endpoints** page, select the endpoint you created.
-1.  Click **Actions**.
-1.  Click **Modify Private DNS Names**.
-1.  Check the **Enable Private DNS Name** checkbox.
-1.  Click **Modify Private DNS Name**.
+1. Click **Endpoints** in the sidebar.
+1. Click **Create Endpoint**.
+1. On the **Create Endpoint** page, for the **Service Category** field, select **Find service by name**.
+1. In the **Service Name** field, enter the **Service Name** copied from the connection modal in [Step 1](#step-1).
+1. Click **Verify**.
+1. In the **VPC** field, enter the ID of the VPC you want to create your endpoint in.
+1. Verify that the subnets are pre-populated.
+1. In the **Security group** section, select the security group you created in [Step 8](#step-8) and uncheck the box for **default** security group.
+1. Click **Create Endpoint**.
 
-Alternatively, use the AWS CLI to modify the Private DNS Name:
+    The VPC Endpoint ID displays.
 
-1.  After the endpoint status changes to Available, run the following AWS CLI command:
+1. Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** modal.
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="aws-cli">
+
+1. Substitute the values from the previous steps and run the following AWS CLI command:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    $ aws ec2 modify-vpc-endpoint --region $REGION \
-    --private-dns-enabled --vpc-endpoint-id $VPC_ENDPOINT_ID
+    aws ec2 create-vpc-endpoint --region {REGION} \
+                                  --vpc-id {VPC_ID} \
+                                  --subnet-ids {SUBNET_ID1} {SUBNET_ID2} \
+                                  --vpc-endpoint-type Interface \
+                                  --security-group-ids {SECURITY_GROUP_ID1} {SECURITY_GROUP_ID2} \
+                                  --service-name {SERVICE_NAME_PROVIDED_BY_COCKROACH}
     ~~~
 
+1. Locate the VPC Endpoint ID in the CLI output.
+
+1. Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** modal.
+
+</section>
+
+</section>
+
+## Step 3. Verify the endpoint ID
```

Comment:
Should this also have steps for Serverless? It's empty here: https://deploy-preview-17461--cockroachdb-docs.netlify.app/docs/cockroachcloud/aws-privatelink?filters=serverless#step-3-verify-the-endpoint-id

Reply:
Done by correcting line 225.